components/pager/events.md

```diff
@@ -36,19 +35,52 @@ The `PageChanged` event fires a new page is selected. You can use it to implemen
     public int ButtonCount { get; set; } = 4;
     public int ItemsOnPage { get; set; } = 10;
     public int CurrentPage { get; set; } = 2;
-    public string Result { get; set; } = String.Empty;
+    public string Result { get; set; }
 
-    void PageChangedHandler(int page)
+    void PageChangedHandler(int newPage)
     {
-        CurrentPage = page;
-        Result = $"Current page: {page}";
+        CurrentPage = newPage;
+        Result = $"Current page: {newPage}";
     }
 }
 ````
 >caption The result from the code snippet above
 
 ![config of the pager with one-way binding](images/pager-data-binding.gif)
 
+## PageSizeChanged
```

Comment:
Did the grid also get a PageSizeChanged event?

Also, I recommend adding a list of the available events at the beginning of the article to ensure people see what is available.

Reply:
Yes, the Grid and ListView-related task is separate.

Will add an event list on top.